dash/include/dash/GlobPtr.h

```diff
@@ -425,6 +425,10 @@ class GlobPtr {
     m_dart_pointer.unitid = unit_id.id;
   }
 
+  constexpr auto get_unit() const noexcept {
```

Comment:
Why not just `unit()`? As in `Team::unit()`?

Reply:
I prefer your suggestion as well, but...`set_unit` already existed. So I decided this to be consistent. We can rename it, but this will certainly impact more source files.

Reply:
Any idea what `set_unit` is used for? Conceptually, it exposes a low-level detail (one element of the tuple `[unit, segment, offset]`) to change from the outside, without touching the segment ID or the offset, which leaves you in a potentially inconsistent state. Maybe we should get rid of `set_unit` entirely?

Reply:
Well semantically it is valid to change the unit of a particular segment. Given a `GlobPtr` to the beginning of a segment we can always "jump" to the next unit. We use it exactly for this.

On the other hand it enables to modify the pointer to point to Mordor leaving you alone in hell. But this is what we can do in good old C as well.